lighthouse-treemap/app/src/main.js

```diff
@@ -307,11 +312,106 @@ class TreemapViewer {
     };
   }
 
+  createTable() {
+    const tableEl = TreemapUtil.find('.lh-table');
+    tableEl.innerHTML = '';
+
+    /** @type {Array<{name: string, resourceBytes: number, unusedBytes?: number}>} */
+    const data = [];
+    TreemapUtil.walk(this.currentTreemapRoot, (node, path) => {
+      if (node.children) return;
+
+      // Elide the first path component, which is common to all nodes.
+      let name;
+      if (path[0] === this.currentTreemapRoot.name) {
+        name = path.slice(1).join('/');
+      } else {
+        name = path.join('/');
+      }
+
+      // Elide the document URL.
+      if (name.startsWith(this.currentTreemapRoot.name)) {
+        name = name.replace(this.currentTreemapRoot.name, '//');
+      }
+
+      data.push({
+        name,
+        resourceBytes: node.resourceBytes,
+        unusedBytes: node.unusedBytes,
+      });
+    });
+
+    /** @param {Tabulator.CellComponent} cell */
+    const makeCoverageTooltip = (cell) => {
+      /** @type {typeof data[number]} */
+      const dataRow = cell.getRow().getData();
+      if (!dataRow.unusedBytes) return '';
+
+      const percent = Math.floor(100 * dataRow.unusedBytes / dataRow.resourceBytes);
+      return `${percent}% bytes unused`;
+    };
+
+    const gridEl = document.createElement('div');
+    tableEl.append(gridEl);
+
+    const maxSize = this.currentTreemapRoot.resourceBytes;
+    this.table = new Tabulator(gridEl, {
+      data,
+      height: '100%',
+      layout: 'fitColumns',
+      tooltips: true,
+      addRowPos: 'top',
+      history: true,
+      resizableColumns: true,
+      initialSort: [
+        {column: 'resourceBytes', dir: 'desc'},
```

Comment:
This is a good initial sort but if I switch to sort by "Unused" or "Size" it's in ascending order first. I have to click again to get the order to be descending.

Is there a way to have descending order be the first option when clicking a column header?

Reply:
I noticed this too. Notice the arrow for the Coverage column is highlighted. I guess it is selecting the last column that matched the "field" of the initial sort. I only used `resourceBytes` as the coverage field to sort by that value. I tried your other suggestion of sorting by percent unused, but the results weren't too useful (small modules dominated either end of the sort). Instead I opted to disable sorting for the coverage column.

> Is there a way to have descending order be the first option when clicking a column header?

Yes, done.